ci/test/pipeline.template.yml

```diff
@@ -43,6 +43,20 @@ steps:
         agents:
           queue: builder-linux-x86_64
 
+      - id: rust-upload-debug-symbols-x86_64
```

Comment:
Is this sufficient in this place or do we need this in all pipelines?

Reply:
I think just here is good!